airbyte-integrations/connectors/source-pinterest/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 5cb7e5fe-38c2-11ec-8d3d-0242ac130003
-  dockerImageTag: 1.3.0
+  dockerImageTag: 1.3.1
```

Comment:
should we bump minor version instead of patch?
as we enable feature and not fix something

Reply:
This one is actually a bug fix, because we were already emitting record counts, but this fixes an issue w/ the record count logic, for some reason only a few sources were affected.